src/worker/index.ts

```diff
@@ -0,0 +1,55 @@
+import ipc from "../ipc";
+import { createTestplaneWorkerRPC } from "./rpc";
+import { getTestplaneAPI } from "./testplane";
+import { TestplaneReporter } from "./reporter";
+import { MasterEvents, type MasterInitEventMessage } from "../types";
+import { WorkerEvents } from "./types";
+
+import type Testplane from "testplane";
+
+type MessageWithError = {
+    e: Error;
+};
+
+ipc.on<MasterInitEventMessage>(MasterEvents.INIT, message => {
+    try {
+        const { cwd, configPath } = message;
+        const testplaneNodePath = require.resolve("testplane", { paths: [cwd] });
```

Comment:
do you remember about yarn and its custom paths? :)

Reply:
I forgot to fix it here, thanks